default/python/AI/freeorion_tools/charts_handler.py

```diff
@@ -15,14 +15,14 @@ def charting_text():
     planet = None
     if planet_id is not None:
         planet = universe.getPlanet(planet_id)
-    aggression_name = fo.aggression.values[foAI.foAIstate.aggression].name
+    aggression_name_local = aggression_name[foAI.foAIstate.character]
```

Comment:
1. I will gladly reword it, because I can only write/understand a few rudimentary sentences in languages other than English.
2. The correct expression is "forced idiom".  I mistyped. Idiom is the style of way of expressing an idea in a language.
3. Perhaps we should do a day in each devs mother tongue to put the English in their place. 😃 

My style for this code was unnatural,  inelegant and awkward.  Using something that looks like an array `table[character]` when `character` is not obviously an index is the wrong approach.  As you pointed out a function is the more natural choice.

I switched to a public function  `get_trait_name_aggression(character)` and expanded the module comments.